### DISCUSSION TITLE
Making an Elevation GeoTIFF from ICESAT Data

### DISCUSSION THREAD

**Discussion by henryolive01** [icesat2py/icepyx#483] [Q&A] [answered]
I want to extract ICESAT2 elevation data (was using ATL03) into a GeoTIFF raster for use in QGIS. I downloaded data from my region and extracted the data in gt1l/heights/h_ph. I used lat_ph and lon_ph for my x and y coordinates. I was more or less left with a 2d line with deviation laterally of ~70cm. Is there a way I can download gridded data or is the best I can do download every track from the h5 file and any other files that were also downloaded as part of my search in my region?
![image](https://github.com/icesat2py/icepyx/assets/154494717/091ea309-5c68-4ab9-bacc-222e7525ac6f)

**Comment by JessicaS11** [comment, marked as answer]
Hello @henryolive01! Thanks for your patience - despite my settings, I haven't gotten notifications for new discussion posts to know they're here.

The answer to your question is an ambiguous "it depends". There are a number of higher level data products that provide heights (some of them gridded) optimized for a variety of surface cover types. NSIDC has a [full list](https://icesat-2.gsfc.nasa.gov/science/data-products) of all the available products (the gridded ones have higher numbers). Whether or not any of them work for you depends on your specific needs as far as temporal and spatial resolution and geographic region/surface cover type. There are a variety of tools (including icepyx) that can help streamline your workflow if you do opt to pursue the photon-level processing you described in your post. icepyx can also get any of the other data products by changing the the `product="ATL03"` portion of your code to whichever product you are interested in. If you share some more information on your objectives, @mikala-nsidc at NSIDC or I should be able to provide more specific suggestions.

**Reply by henryolive01** [reply]
Hi Jessica,

Thank you for your reply. I have had a look at the list of available data products. It appears ATL18 is the one I'm after however it's been removed. 
My goal was to find gridded elevation data to model rainfall runoff. There is other data available to 30m resolution and I was hoping ICESAT2 data could be used and at a higher resolution. Thanks again for your reply in clarifying this for me.

**Reply by mikala-nsidc** [reply]
Hi Henry, 
Chiming in from NSIDC.  ATL18 is still in development by the scientists and developers working on it.  We do expect to host it once it's ready.  But it could be some time before we get it at NSIDC.  Have you tried SlideRule yet?  Using the ATL08 algorithms within SlideRule may get you part of the way there (geopandas data frame), while you wait for the ready-to-go ATL18 gridded canopy and terrain elevation product. 
Always feel free to reach out to us at nsidc@nsidc.org with questions regarding ICESat-2 data.  
Kind regards,
Mikala
User Services at NSIDC

**Reply by henryolive01** [reply]
Thank you :))

**Reply by mikala-nsidc** [reply]
I'm realizing I made an assumption about your first name!  Apologies @henryolive01 in case that is incorrect!
